Mathlib/LinearAlgebra/Matrix/DotProduct.lean

```diff
@@ -101,17 +102,27 @@ section StarOrderedRing
 
 variable [PartialOrder R] [NonUnitalRing R] [StarRing R] [StarOrderedRing R] [NoZeroDivisors R]
 
+/-- Note that this applies to `ℂ` via `Complex.strictOrderedCommRing`. -/
```

Comment:
It looks like you copied all these docstrings from the few declarations below, but they're all wrong? I think they apply to `ℂ` via `RCLike.toStarOrderedRing`, which is available in the `ComplexOrder` scope.

Reply:
I think this was accurate at the time, but is no longer.